benchmarks/backend_request_func.py

```diff
@@ -285,9 +287,14 @@ async def async_request_openai_completions(
 
                                 most_recent_timestamp = timestamp
                                 generated_text += data["choices"][0]["text"]
-
+                    if first_chunk_received:
```

Comment:
This will give `UnboundLocalError: cannot access local variable \'first_chunk_received\' where it is not associated with a value` when no chunk is ever received.

Reply:
Nice catch, I have opened another PR to fix it.